index.js

```diff
@@ -1,11 +1,50 @@
 'use strict';
 
 /**
- * Adds commas to a number
- * @param {number} number
- * @param {string} locale
+ * Push and pop from a stack
+ * @param {string} value
+ * @param {number} value
  * @return {string}
+ * @return {array}
  */
-module.exports = function(number, locale) {
-    return number.toLocaleString(locale);
-};
+
+//  creating a stack of length 5
+let lengthOfStack = 5
+let stack = new Array(lengthOfStack);
+let lastUpdatedIndex = 0;
+
+let addValueToStack = (value) => {
+// pushing value onto the stack
+    if (lastUpdatedIndex === lengthOfStack){
+        return "Stack is full, cannot push."
+    }
+    else {
```

Comment:
if possible try to decrease "code indentation" so the code that is written like so:
```
if () {
}
else {
}

```

can be re-written as 

```
if () {
 return
}

do something
return 
```

Reply:
Code revised to incorporate feedback.